controller/lib/core/blower_pid.cpp

```diff
@@ -27,28 +27,17 @@ static double Input;
 static double Output;
 
 // PID-tuning were chosen by following the Ziegler-Nichols method,
-// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method.
-//
-// Here's an achieved pressure vs. desired pressure trace collected on Alpha
-// build, 2020-05-01, with the following settings.
-//
-//   PEEP = 10 cm H2O,
-//   PIP = 30 cm H2O,
-//   RR = 10 breaths/min
-//   I:E = 2
-//
-// https://drive.google.com/file/d/13QyvO1ptUYYUAs8qT0dCyW9f1XZ8w7iw
+// https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method
 //
 // Note that Ku and Tu only seem to work with this particular sample time.
 static constexpr Duration PID_SAMPLE_TIME = milliseconds(10);
-static constexpr float Ku = 600;
-static constexpr Duration Tu = seconds(2);
-
-// "Classic PID" settings.  Wikipedia says to expect aggressive overshoot with
-// these params, but I don't observe any overshoot.
-static constexpr float Kp = 0.6 * Ku;
-static constexpr float Ki = 1.2 * Ku / Tu.seconds();
-static constexpr float Kd = 3 * Ku * Tu.seconds() / 40;
+static constexpr float Ku = 200;
+static constexpr Duration Tu = seconds(1.5);
+
+// "No overshoot" settings.
```

Comment:
Explain why "no overshoot" is the mode we need?

Reply:
Done